src/main/resources/templates/progress/teamDetail.html

```diff
@@ -22,9 +22,9 @@
             <h1>Tým</h1>
             <p class="subheading" th:text="${team.name}"></p>
 
-            <div th:each="cypher : ${cyphers}" class="team">
+            <div th:each="cypher : ${cyphers}" class="cypher organizer">
                 <th:block th:with="status=${cyphersStatuses.get(cypher.cypherId)}">
-                    <div class="row">
+                    <div class="row align-items-center">
                         <div class="col-12">
                             <p th:classappend="${#strings.toLowerCase(status)}">
```

Comment:
Toto `th:classappend="${#strings.toLowerCase(status)}"` presun prosimte na ten `<div class="col-12">` a ten element <p> odstran uplne. Pak to bude dokonale ;)

Reply:
hotovo :)